samples/boot/oauth2resourceserver-multitenancy/README.adoc

```diff
@@ -23,13 +23,21 @@ By default, the tests are pointing at a mock Authorization Server instance.
 The tests are configured with a set of hard-coded tokens originally obtained from the mock Authorization Server,
 and each makes a query to the Resource Server with their corresponding token.
 
-The Resource Server subsquently verifies with the Authorization Server and authorizes the request, returning the phrase
+The Resource Server subsequently verifies with the Authorization Server and authorizes the request, returning either the
+phrase
 
 ```bash
-Hello, subject!
+Hello, subject for tenantOne!
 ```
 
-where "subject" is the value of the `sub` field in the JWT returned by the Authorization Server.
+where "subject" is the value of the `sub` field in the JWT sent in the `Authorization` header,
+
+or the phrase
+```bash
+Hello, subject for tenantTwo!
+```
+where "subject" is the value of the `sub` field in the response from the Authorization Server.
```

Comment:
Could we add a comment here about it being introspection?

```suggestion
where "subject" is the value of the `sub` field in the Introspection response from the Authorization Server.
```

Reply:
Done!